svg.js.d.ts

```diff
@@ -1078,6 +1139,7 @@ declare module "@svgdotjs/svg.js" {
     class Svg extends Container {
         constructor(svgElement?: SVGSVGElement);
         constructor(id: string);
+        constructor(domElement: HTMLElement);

```

Comment:
This one is not true. The `<svg>` constructor can only take a node or an attributes array. Dont mix this up with the `SVG()` method which can take all these.

Reply:
@Fuzzyma Yep, you're right the constructor should different from SVG().